### ISSUE TITLE
Need the ability to update the counter to a new value

### ISSUE THREAD

**Issue by gokulwm** [gokulwm/lab-agile-planning#7]
**As a** User  
 **I need** to start the counter from a new value  
 **So that** i can count the correct things from some count of its which is previously known  
   
 ### Details and Assumptions any details helpful for the developers to build the story
 * A button with naming update is to be placed
 ### Acceptance Criteria  
   
 ```gherkin
 Given i have my counter value as 5
 When i click the update button, it asks me to enter the new value
 Then i should be able to retrieve and look into the modified counter value
 ```
